.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Release
+
+on:
+  push:
+    tags:
```

Comment:
To facilitate testing this workflow in GitHub Actions, this used to listen to pushes on a branch. Before opening the PR, I updated it specifically to listen to pushes on tags.